### ISSUE TITLE
Add option to convert between wxMenuToolbar and wxMenu (PopupMenu)

### ISSUE THREAD

**Issue by KeyWorksRW** [KeyWorksRW/wxUiEditor#688]
### Description:

When importing a Windows Resource file, there isn't a reliable way to know if a menu resource will be used as a menu bar or as a popup menu. Currently, we take an educated guess, but we're wrong some of the time, and switching between the types is a pain. One possibility would be for our navigation menu to check a wxMenuBar and if it only has one child wxMenu, then we could offer the option to convert it to a popup menu. Converting the other direction would also be possible, though we would need to create a label for the wxMenu.

**Comment by Randalphwa** [comment]
While converting from a Windows Resource to a wxWidgets UI mostly works, the code to support the UI requires quite a bit of work and a fairly good knowledge of how wxWidgets works. It gets even more difficult if the app was written using MFC. Given that I have yet to receive any feedback about the Windows Resource conversion (which is _not_ perfect by any means) I suspect no one is using wxUiEditor to do the conversion. Either that, or they tried it, realized how much more work it would take to change all their code, and decided it wasn't worth it.

Bottom line, the only reasonable use case I can think of for the menu type change is the Windows Resource conversion. It's not worth spending time on in the off chance someone might use it someday...
